src/main/java/ec/edu/ups/model/Cita.java

```diff
@@ -0,0 +1,72 @@
+package ec.edu.ups.model;
+
+import java.io.Serializable;
+import java.time.LocalTime;
+
+public class Cita implements Serializable {
+	
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
+	private int idCitas;
```

Comment:
Un int forma parte de un Primary Data Type en Java, por lo tanto, si llegamos a ciertos números de ingresos en un modelo se va a romper, para la OOP se utiliza Integer.
https://www.tutorialspoint.com/difference-between-an-integer-and-int-in-java